### DISCUSSION TITLE
Beginner in GSOC

### DISCUSSION THREAD

**Discussion by Avdhesh-Varshney** [MDAnalysis/mdanalysis#4462] [GSoC Discussions]
Hello coders, Happy to connect with you all.
My name is Avdhesh Varshney. I am a pre-final year B Tech student at Dr. B R Ambedkar NIT Jalandhar interested in Python, C++, JS, ML, DL, NLP, GenAI, front-end web development. I am interested in this field and have worked on some few projects in these areas involving ML/AI and python backend with Django, Flask and Streamlit.

LinkedIn - https://www.linkedin.com/in/avdhesh-varshney/
GitHub - https://github.com/Avdhesh-Varshney

I look forward to participating in GSOC 2024. If possible, please let me know if how to contribute as a beginner to make this journey smooth. Also to get selected as a GSOC project contributor which steps should be taken and how to begin?

**Comment by jennaswa** [comment]
MDAnalysis is a Python package for analyzing molecular simulations data. We have a few [introductory videos](https://www.mdanalysis.org/pages/learning_MDAnalysis/#introductory) that can give you an idea of what problems MDAnalysis is solving and help you to start [learning MDAnalysis](https://www.mdanalysis.org/pages/learning_MDAnalysis/). If you haven't used MDAnalysis before, the best way to get started is to get your hands dirty following the [MDAnalysis User Guide](https://userguide.mdanalysis.org/stable/index.html). We suggest starting with [installing](https://userguide.mdanalysis.org/stable/installation.html) the MDAnalysis package and going through the [Quick Start Guide](https://userguide.mdanalysis.org/stable/examples/quickstart.html). Other sections of the User Guide go into many more details about the different capabilities of MDAnalysis.

Hopefully this will help you get started, but please don't hesitate to reach out if you have any questions.
